test/common/markdown/markdown-spell.tests.ps1

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Verify Markdown Spelling" {
+    BeforeAll {
+        if(!(Get-Command -Name 'mdspell' -ErrorAction SilentlyContinue)) {
+            start-nativeExecution {
+                sudo npm install -g markdown-spellcheck@0.11.0
+            }
+        }
+
+        # Cleanup jobs for reliability
+        get-job | remove-job -force
+    }
+
+    AfterAll {
+        # Cleanup jobs to leave the process the same
+        get-job | remove-job -force
+    }
+
+    $groups = Get-ChildItem -Path "$PSScriptRoot\..\..\..\*.md" -Recurse | 
+        Where-Object DirectoryName -notlike '*node_modules*' | 
+        Group-Object -Property directory
+
+    $jobs = @{}
+    # Start all spelling verification in parallel
+    Foreach($group in $groups)
+    {
+        Write-Verbose -verbose "starting jobs for $($group.Name) ..."
+        $job = Start-ThreadJob {
+            param([object[]] $group)
+            foreach($file in $group)
+            {
+                $results = mdspell --en-us --ignore-numbers --ignore-acronyms --report $file 2>&1 '!CHANGELOG.md'
+                Write-Output ([PSCustomObject]@{
+                    file = $file
+                    results = $results
+                })
+            }
+        } -ArgumentList @($group.Group)
+        $jobs.add($group.name,$job)
+    }
+
+    # Get the results and verify
+    foreach($key in $jobs.GetEnumerator().Name)
+    {
+        $job = $jobs.$key
+        $results = Receive-Job -Job $job -Wait
+        Remove-job -job $Job
+        foreach($jobResult in $results)
+        {
+            $file = $jobResult.file
+            $result = $jobResult.results
+            Context "Verify spelling in $file" {
+                $failures = @($result) -like '*spelling errors found in*'
+                $passes = @($result) -like '*free of spelling*'
+ 
+                $trueFailures = foreach ($Failure in $Failures) {
+                    @{ Spell = $Failure }
+                }
+
+                if($trueFailures) {
+                    it "<spell> should work" -TestCases $trueFailures {
+                        param($spell)
+                        throw "Tool reported spelling as wrong."
```

Comment:
If we want to see spell errors we could output its here. 
Replace It-block with Should

Reply:
vexx32 seems to have it under control

Reply:
```suggestion
                        throw "Tool reported spelling as wrong.`n$spell"
```

Reply:
If $trueFailures contains many values we get many lines "Tool reported spelling as wrong".